docs/platform/what-is-tizen/versions/tizen-6-5-m2.md

```diff
@@ -0,0 +1,137 @@
+# Tizen 6.5
```

Comment:
Since this is a new topic, please update the toc_all.md file to include this file in Left Nav

Reply:
I updated toc_all.md file for adding Tizen 6.5